h/views/admin_groups.py

```diff
@@ -12,12 +12,29 @@
 from h.models.annotation import Annotation
 from h.models.group import GroupFactory
 from h.models.group_scope import GroupScope
+from h.models.organization import Organization
 from h import paginator
-from h.schemas.admin_group import CreateAdminGroupSchema, user_exists_validator_factory
+from h.schemas.admin_group import CreateAdminGroupSchema
 
 _ = i18n.TranslationString
 
 
+def _find_organization(session, pubid):
```

Comment:
View logic in general throughout the app seems thicker than I'm used to (just a general observation). I wonder if places like here could benefit from `context`-layer stuff in the future...just a thought.